packages/wallet/backend/src/quote/service.ts

```diff
@@ -255,10 +259,14 @@ export class QuoteService implements IQuoteService {
       paymentPointerId: senderPpId,
       amount,
       description: 'Currency exchange.',
-      isReceive: false,
+      isReceive: true,
       receiver: receiverPpUrl
     })
 
+    //OUTCOME
+    Node_Cache.set(quote.id, quote)
```

Comment:
as the quote is not used and only helps to check if it is an exchange payment, cache it with a `true` value (or empty object if it requires the value to be an object)